src/data/set/intervals.lean

```diff
@@ -189,6 +214,18 @@ lemma Icc_subset_Ici_iff (h₁ : a₁ ≤ b₁) :
   Icc a₁ b₁ ⊆ Ici a₂ ↔ a₂ ≤ a₁ :=
 ⟨λ h, h ⟨le_refl _, h₁⟩, λ h x ⟨hx, hx'⟩, le_trans h hx⟩
 
+lemma Ioi_subset_Ioi (h : a ≤ b) : Ioi b ⊆ Ioi a :=
```

Comment:
Should these be iffs?

Reply:
In a preorder, this is not an equivalence. For instance, if you have three elements with `a < b`, `a'< b` and `a` not comparable to `a'`, then `Ioi a = Ioi a'`.

Reply:
Ah, sneaky. Should we have the lemma as an equivalence, or even a simp lemma, under the conditions where it holds? Doesn't have to be in this PR, we can add it when it's needed I guess.

Reply:
I have added the iff versions in the section on linear orders, as simp lemmas.